code/style.css

```diff
@@ -1,4 +1,188 @@
-html {
-  background: #0025ff;
-  color: #fff;
+
+a {
+  font-family: "helvetica neue";
+  text-decoration: none;
+  color: inherit;
 }
+
+a:hover {
+  border-bottom: 3px dotted #ff7a59;
+}
+
+/* styling for header and navbar */
+
+header {
+  background: white;
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 10px;
+}
+
+header img {
+  height: 50px;
+  width: 50px;
```

Comment:
We should avoid setting both width and height of `img` tags, since because of that we manipulate proportions of image. If you will uncheck width proerty in devtools, you will know what I mean